index.html

```diff
@@ -192,11 +192,13 @@ <h1 class="intro">About Myself</h1>
                 <h3>Languages</h3>
               </div>
               <div>
-                <img
-                  class="icon"
-                  src="./Assets/dropdown-icon.png"
-                  alt="drop_down_icon"
-                />
+                <a href="#">
```

Comment:
- [ ] **Required:** The anchor tag or the <a> tag, is one of the most important [HTML tags in terms of SEO](https://en.ryte.com/wiki/Anchor_Tag). This is because of the navigation abilities. You are not passing this test because these tags all point to your root `a href="#"`, kindly remove the tags since they don't point anywhere.